app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -386,7 +388,7 @@ public void onDestroy() {
         disposables.clear();
         positionSubscriber = null;
         currentWorker = null;
-        bottomSheetBehavior.setBottomSheetCallback(null);
+        bottomSheetBehavior.removeBottomSheetCallback(bottomSheetCallback);
```

Comment:
Are you sure nothing bad happens if `bottomSheetCallback` is `null` or it has already been removed?

Reply:
The callbacks are stored in an `ArrayList`, and the method calls the list's remove method.

Just in case, I'll check.